### PR TITLE
Updates related to ArgoCD in OCP GitOps Documentation

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.7` and enterprise-`4.8`
- **JIRA issues**: [RHDEVDOCS-2982](https://issues.redhat.com/browse/RHDEVDOCS-2982),  [RHDEVDOCS-2959](https://issues.redhat.com/browse/RHDEVDOCS-2959)
- **Preview pages**: [Configuring SSO for Argo CD on OpenShift](https://deploy-preview-32787--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-sso-for-argo-cd-on-openshift.html), [Installing GitOps Operator in web console](https://deploy-preview-32787--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/installing-openshift-gitops.html#installing-gitops-operator-in-web-console_getting-started-with-openshift-gitops)
- **Reviewers**: Ruchir Garg, Dewan Ahmed, Preeti Chandrashekar

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@rgarg1 @dewan-ahmed @Preeticp PTAL and kindly provide comments/LGTM. Thanks!

File: cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::modules/gitops-document-attributes.adoc[]
 
 toc::[]
 
-Once the Red Hat OpenShift GitOps Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO. You can use Keycloak or a bundled Dex OIDC provider.
+Once the Red Hat OpenShift GitOps Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO.
```
Comment (rgarg1) [inline cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc:9]: Should we mention that Dex is unsupported here, or it's already captured elsewhere?

**Review by Preeticp — COMMENTED** [review]
@Srivaralakshmi thanks for the PR, I have added a few suggestions, PTAL.

File: cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc:9]: Sure, will do.

**Review by Preeticp — COMMENTED** [review]
@Srivaralakshmi thanks for the PR, I have added a few suggestions, PTAL.

File: modules/installing-gitops-operator-in-web-console.adoc [hunk 2]
```diff
@@ -5,10 +5,19 @@
 [id="installing-gitops-operator-in-web-console_{context}"]
 = Installing GitOps Operator in web console
 
+By default the Operator is installed in the `openshift-operators` project.
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/installing-gitops-operator-in-web-console.adoc:8]: @Preeticp Sure, will do! 

FYI, @Dewan has responded about Line 27, but we are also awaiting confirmation from @wtam2018 and @amitkrout.

[the remark this answers is not in the document]
Comment (wtam2018) [reply modules/installing-gitops-operator-in-web-console.adoc:8]: OpenShift Operator is installed in all namespaces.

File: cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc:9]: @rgarg1 I do not see any mention of Dex supported/not supported anywhere else.

@wtam2018 Should we mention that Dex is unsupported here, WDYT? Thanks!

[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc:9]: Done.

File: modules/installing-gitops-operator-in-web-console.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/installing-gitops-operator-in-web-console.adoc:8]: Done.

**Comment by Srivaralakshmi** [comment]
@rgarg1 @dewan-ahmed @Preeticp I have incorporated all the review comments received so far. PTAL and kindly provide comments/LGTM. Thanks!

File: cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc (hunk 1 above)
Comment (Preeticp) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc]: @siamaksade @sbose78 can you let us know your thoughts on this, please?

**Comment by Preeticp** [comment]
Apart from the question about Dex, LGTM from my side, @Srivaralakshmi

File: cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc (hunk 1 above)
Comment (wtam2018) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc]: I have we should mention that Dex is unsupported.

Comment (Srivaralakshmi) [reply cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc]: Done.

**Comment by Srivaralakshmi** [comment]
@rgarg1 @dewan-ahmed @Preeticp I have fixed all the comments. PTAL and kindly provide LGTM. Thanks!

**Review by dewan-ahmed — APPROVED** [review]
/lgtm

**Pull request merged**